src/gtk/menu.cpp

```diff
@@ -762,6 +766,36 @@ void wxMenuItem::SetBitmap(const wxBitmap& bitmap)
     }
 }
 
+wxBitmap wxMenuItem::GetBitmap() const
+{
+    wxBitmap bmp;
+    if ( m_bitmap.IsOk() )
+    {
+        if ( m_parentMenu && m_parentMenu->GetWindow() )
+        {
+            bmp = m_bitmap.GetBitmapFor(m_parentMenu->GetWindow());
+        }
+        else
+        {
+            bmp = m_bitmap.GetBitmap(wxDefaultSize);
```

Comment:
In which case can we get here, i.e. when do we need a bitmap for a menu item without any window?

Reply:
I'd still like to know the answer to this.

Reply:
It can be called from GtkAppend while wxMenu doesn't attached to wxMenubar yet.
Also, it is a public function, so it can be called directly even before wxMenuItem will be attached to wxMenu.